ecosystem/sep-0006.md

```diff
@@ -168,6 +168,7 @@ The response body should be a JSON object with the following fields:
 Name | Type | Description
 -----|------|------------
 `how` | string | Terse but complete instructions for how to deposit the asset. In the case of most cryptocurrencies it is just an address to which the deposit should be sent.
+`id` | string | (optional) The anchor's internal ID for this deposit request. The wallet will use this ID to query the [`/transaction`](#single-historical-transaction) endpoint to check status of the request.
```

Comment:
💡  Internal ID implies to me it is an ID internal to the anchor that would not be exposed publicly, but in this case it is being exposed to another party. Stating that it is the anchor's ID for the deposit is sufficient.
```suggestion
`id` | string | (optional) The anchor's ID for this deposit. The wallet will use this ID to query the [`/transaction`](#single-historical-transaction) endpoint to check status of the request.
```